### ISSUE TITLE
Vercel Deployment Issue :

### ISSUE THREAD

**Issue by PRITHIVSAKTHIUR** [Jasiri-w/Sportshall-Registration-Web-App#8]
"My issue was with dependencies. I used a brute force ground-up approach to figure this out by deleting my requirements.txt file and reinstalling all the necessary packages one by one in an isolated virtual environment." - by you.

Bro you have mentioned in the vercel Docs, the solution for 250MB size Exceeded in Serverless Function. 

You had also mentioned that it can be cleared by installing package Dependencies in Vercel (.venv)  Virtual Environment one by one, Can you please explain How it can be Done ?.

If you respond, It will be Helpful to me. Thank you.

**Comment by Jasiri-w** [comment]
Hi Prithiv,

If you read that comment closely, you'll see my exact solution. My problem was that the full size of the dependencies installed was obese. This was because the requirements.txt file I had generated using `pip freeze` included all global packages installed on my local computer as opposed to the modules strictly used in my project.

That being said, I first deleted my existing obese requirements file. I created a new virtual environment for my project with no extra modules installed. This obviously created errors as the Django project would require numerous modules, so from there, I reinstalled strictly the required modules that would solve each error one by one locally so that once I am complete and freeze my new pip module list it will be a lean list only including required packages and therefore reducing the module folder when deployed on Vercel.

I hope this helps
